packages/fab/src/mwc-fab.ts

```diff
@@ -0,0 +1,81 @@
+/**
+@license
+Copyright 2018 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import {LitElement, html, property, customElement} from '@polymer/lit-element/lit-element.js';
+import {classMap} from 'lit-html/directives/classMap.js';
+import {style} from './mwc-fab-css';
+import {ripple} from '@material/mwc-ripple/ripple-directive.js';
+import '@material/mwc-icon/mwc-icon-font.js';
+
+@customElement('mwc-fab' as any)
+export class Fab extends LitElement {
+
+  @property({type: Boolean})
+  mini = false;
+
+  @property({type: Boolean})
+  exited = false;
+
+  @property({type: Boolean})
+  disabled = false;
+
+  @property({type: Boolean})
+  extended = false;
+
+  @property({type: Boolean})
+  showIconAtEnd = false;
```

Comment:
won't this look weird as attribute? `showiconatend` - that is really hard to read

Reply:
This isn't reflected by default so shouldn't be a problem for now